tool-testsuite/test/org/antlr/v4/test/tool/TestVocabulary.java

```diff
@@ -32,10 +32,13 @@
 import org.antlr.v4.runtime.Token;
 import org.antlr.v4.runtime.Vocabulary;
 import org.antlr.v4.runtime.VocabularyImpl;
+import org.antlr.v4.tool.Grammar;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 
+import static junit.framework.TestCase.assertEquals;
```

Comment:
This import is from the old JUnit 3 framework. Everywhere else we use the one I mentioned.